release.py

```diff
@@ -47,19 +51,73 @@ def release_github(test=True):
             headers={"Authorization": "token " + os.environ["GITHUB_TOKEN"]})
         print(response.text)
 
+    # Create a Matlab toolbox and upload it
+    pull_light_wrapper()
+    create_mltbx()
```

Comment:
Perhaps this should be part of the CI process too, so we get a toolbox build artefact for every (Linux) build. It could definitely be useful for debugging, and let us check things as we go along rather than waiting for a release.

Reply:
Oh ok... I was just going to wait until I get admin and to setup a "release" pipeline. But yes, it might be good just to generate a `mltbx` file for every build (they're not big, only about 21 kb).

Reply:
Ah sorry, I forgot about that! That can be done in a separate PR later then (or I'll do it if you still don't have permissions by then...)